oci.sh

```diff
@@ -11,7 +11,7 @@ source $(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)/lib/commo
 
 greenprint "🔮☁ Starting OCI cleanup"
 
-if ! hash oci; then
+if ! hash /root/bin/oci; then
```

Comment:
This won't work, it expects a command name it seems... I tried it locally with `hash /bin/bash` for example and in the cloud-tools container as well and it fails... Maybe changin this to like `$OCI_CMD --version` ?